relation_engine_server/utils/pull_spec.py

```diff
@@ -125,7 +122,4 @@ def _save_release_id(info):
 
 
 if __name__ == '__main__':
-    if len(sys.argv) > 1:
-        if sys.argv[1] == 'init':
-            download_specs(init_collections=True)
```

Comment:
`init_collections` is true by default, so this just runs the same command twice.